core/src/Cabana_VerletList.hpp

```diff
@@ -518,16 +523,22 @@ struct VerletListBuilder
 
   CSR layout implementation.
 */
-template<class MemorySpace, class AlgorithmTag, class LayoutTag>
+template<class DeviceType, class AlgorithmTag, class LayoutTag>
 class VerletList
 {
   public:
 
+    // Device type.
+    using device_type = DeviceType;
+
     // The memory space in which the neighbor list data resides.
-    using memory_space = MemorySpace;
+    using memory_space = typename device_type::memory_space;
+
+    // Execution space.
+    using execution_space = typename device_type::execution_space;
 
     // Verlet list data.
-    VerletListData<MemorySpace,LayoutTag> _data;
+    VerletListData<memory_space,LayoutTag> _data;
```

Comment:
This change (using type member "alias" instead of the template parameter) is inconsistent with what you do in other classes

Reply:
fixed